src/main/resources/templates/subscription.html

```diff
@@ -166,6 +167,7 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                         </label>
                                     </div>
 
+
```

Comment:
This extra empty line may not be needed.

Reply:
done